### PR TITLE
Hiding the text entry object from the general wiki

### PR DESCRIPTION
**Pull request by HelperWesley**
-Moved all text entry related images, except 1 that was not linked on the page, to a new folder called text_entry
-Renamed all image links on that page to include that new folder
-Renamed text_entry.md to input.md
-Added a .pages file with "hide: true" in it to hide the folder

### CODE REVIEW

**Comment by HelperWesley** [comment]
@Bouh 👋😅

File: docs/gdevelop5/objects/text_entry/text_entry.pages
Comment (Bouh) [inline docs/gdevelop5/objects/text_entry/text_entry.pages:1]: This file should be named `.pages` not `text_entry.pages`

**Review by Bouh — COMMENTED** [review]
`docs/gdevelop5/objects/text_entry/text_entry.pages`

This file should be named `.pages` not `text_entry.pages`

**Comment by HelperWesley** [comment]
@Bouh Changed it. 👍

I assumed the .pages needed the name of the folder, but I guess that was wrong. 😅

Hopefully that's right now.

**Review by Bouh — APPROVED** [review]
Perfect, the page is still accessible with the direct link, but no more listed in the object list.

**Pull request merged**